src/pages/Login/styles.css

```diff
@@ -0,0 +1,67 @@
+.loginUser {
+    height: 100vh;
+    width: 100%;
+    position: relative;
+    overflow: hidden;
+    background-color: var(--background-color);
+}
+.logoImg {
+    padding: 1rem 2rem;
+    display: flex;
+    align-items: center;
+}
+img {
+    height: 3rem;
+    margin-right: 4rem;
+}
+.loginContainer {
+    width: 100vw;
+    height: 100vh;
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    color: white;
+}
+.login-form {
+    border: 1px solid white;
+    width: 30rem;
+    height: 20rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    border-radius: 5px;
+}
+.SignIn-Title {
```

Comment:
you should follow consistent naming convention while giving class names

Reply:
Ok, thanks for review